keycloak/Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM quay.io/keycloak/keycloak:11.0.1
-
+<<<<<<<<< Temporary merge branch 1
```

Comment:
This line would be deleted. It seems to be the comment when being merged.

Reply:
Apologies for this. Was overlooked after syncing our fork. This has been fixed now